cloudify_ansible/utils.py

```diff
@@ -19,9 +19,16 @@
 import yaml
 
 from cloudify.exceptions import NonRecoverableError
+try:
+    from cloudify.constants import RELATIONSHIP_INSTANCE
+except ImportError:
+    RELATIONSHIP_INSTANCE = 'relationship-instance'
 
-BP_INCLUDES_PATH = '/opt/manager/resources/blueprints/' \
-                   '{tenant}/{blueprint}/{relative_path}'
+from cloudify_ansible.constants import (
+    BP_INCLUDES_PATH,
+    WORKSPACE,
+    # SOURCES
```

Comment:
@EarthmanT Do we still need this commented SOURCES

Reply:
Thanks. Yeah I want to use it.